webapp/commonComponents/tooltip.js

```diff
@@ -22,21 +22,16 @@ class Tooltip extends React.Component {
   }
 
   mouseEnter () {
-    const { messages, type = '' } = this.props
-
-    if (!(R.isEmpty(messages) || R.isNil(messages))) {
+    const { messages, content, show, type = '' } = this.props
```

Comment:
can you add show to defaultProps (and specify in a comment how it's used)

Reply:
Ok